.github/workflows/packages.yml

```diff
@@ -0,0 +1,130 @@
+name: Build Packages
```

Comment:
This is a cosmetic thing, but would it make sense to combine this somehow so that there is only one continuous integration workflow listed for each update on [this page](https://github.com/fabiangreffrath/woof/actions)?

Reply:
Nevermind, it's fine to simply bookmark or link this to users: https://github.com/fabiangreffrath/woof/actions/workflows/packages.yml